### PR TITLE
Feat statistics

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary
1. Regular statistical data after entering the system.
2. Separate the parameter `Version` from the operation and maintenance operation requests so that statistics reporting is not aware of when calling these requests.

### CODE REVIEW

File: ui/src/utils/helper.ts [hunk 1]
```diff
@@ -66,3 +72,35 @@ export const formatPatchPoolData = (originUnitData: PoolDetailType,type:'edit'|'
   }
   return newOriginUnitData;
 };
+
+export const getAppInfoFromStorage = async (): Promise<API.AppInfo> => {
```
Comment (powerfooI) [inline ui/src/utils/helper.ts:76]: What if the version changes?

Comment (yang1666204) [reply ui/src/utils/helper.ts:76]: SessionStorage will be cleared when the window is closed or the browser is closed. Do users need to build different packages when using different versions? If it is, there will be no impact

Comment (powerfooI) [reply ui/src/utils/helper.ts:76]: I see, storing in sessionStorage is OK.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**